resources/spec.yaml

```
@@ -11859,6 +11902,11 @@ definitions:
         type: string
     x-okta-tags:
       - UserFactor
+  RawString:
+    type: string
+    x-okta-tags:
+      - Application
+      - IdentityProvider
```

This is not compatible with Java swagger-codegen.
```suggestion
  RawString:
    properties:
      responce:
        type: string
    x-okta-tags:
      - Application
      - IdentityProvider
```

In fact, in a whole file, we do not have such construction, where type defined for the model itself. This is a new behavior.

---

What's the purpose of defining a schema for a raw string in general? Is this a hard limitation of some of our code generation tools?

These endpoints are returning XML. The apps API is more standardized and is correctly returning the right `Content-Type: application/xml;charset=ISO-8859-1` and if we provide a schema, it should actually be a valid schema for XML (unsure about OAS2 support but OAS3 has a pattern for this https://swagger.io/docs/specification/data-models/representing-xml/). The IdP metadata API seems to be very non-standard and looks to be exploiting the behavior of browsers to download the xml file by advertising it as a binary octet-stream.

Probably less relevant now, but the change proposal doesn't seem correct as it would imply some sort of schema with a `response` key.